README.md

```diff
@@ -0,0 +1,49 @@
+## Usage
+### Neovim
+This grammar can be setup within [Neovim](https://github.com/neovim/neovim/) to provide highlighting for the `.inp` files used in [TOPAS-Academic](http://www.topas-academic.net/). 
+It requires:
+- **Neovim 0.10** or newer.
+- Git
+- A C compiler
+1. Install [Nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter) using your preferred package manager. With [vim-plug](https://github.com/junegunn/vim-plug), for example, add this command to your `init.vim`:
+      ```vimscript
+      Plug 'nvim-treesitter/nvim-treesitter', {'do': ':TSUpdate'}
+      ```
+    then restart Neovim and run `:PlugInstall`
+  
+2. Add the following Lua snippet to `init.vim`:
+    ```lua
+    lua << EOF
+    vim.filetype.add {
+      extension = {
+        inp = 'topas',
+      }
+    }
+
+    local parser_config = require "nvim-treesitter.parsers".get_parser_configs()
+    parser_config.topas = {
+      install_info = {
+        url = "https://github.com/JamesDoingStuff/tree-sitter-topas", -- can be replaced with path to cloned repo
+        files = {"src/parser.c"}, 
+        branch = "main",
+        requires_generate_from_grammar = false,
+      },
+      filetype = "inp", 
+    }
+
+    require'nvim-treesitter.configs'.setup{
+      highlight={
+        enable = true,
+        additional_vim_regex_highlighting = false,
+      },
+    }
+    EOF
+    ```
+3. To apply highlights to a file, queries are used to match up nodes of the syntax tree, such as `(identifier)`, to a pre-configured higlight group, such as `variable`.
+
+   The highlights file cannot yet be installed automatically;
+   download [highlights.scm](https://github.com/JamesDoingStuff/tree-sitter-topas/blob/5b15a6531060ce374a17102abb5103b1d8cf04ec/queries/highlights.scm) from this repository and add it to a location within your
```

Comment:
We probably want to point users to the file in the repo, and provide a link to the latest raw instead of a specific commit

Reply:
I've swapped it out for a relative link - is that what you mean? Or would a full link to the file in `main` be better?

Reply:
That link seems fine to me, most `nvim` users will be perfectly capable of getting the raw file from there